watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination.py

```diff
@@ -131,6 +202,39 @@ def set_operating_conditions(m):
     dye_sep.P1.eta_pump.fix(0.75)  # pump efficiency [-]
     dye_sep.P1.lift_height.unfix()
 
+    if hasattr(m.fs, "gac"):
+        m.fs.tb_nf_gac.properties_out[0].temperature.fix(298.15)
+        m.fs.tb_nf_gac.properties_out[0].pressure.fix(101325)
+
+        m.fs.gac.freund_k.fix(10)
+        m.fs.gac.freund_ninv.fix(0.9)
+        m.fs.gac.shape_correction_factor.fix()
+        m.fs.gac.ds.fix(5e-13)
+        # gac particle specifications
+        m.fs.gac.particle_dens_app.fix(750)
+        m.fs.gac.particle_dia.fix(0.001)
+        # adsorber bed specifications
+        m.fs.gac.ebct.fix(600)
+        m.fs.gac.bed_voidage.fix(0.4)
+        m.fs.gac.bed_length.fix(6)
+        # design spec
+        m.fs.gac.conc_ratio_replace.fix(0.50)
+        # parameters
+        m.fs.gac.a0.fix(3.68421)
```

Comment:
Most of the `m.fs.gac` parameters are unchanged from the values set in https://github.com/watertap-org/watertap/blob/main/watertap/examples/flowsheets/gac/gac.py since we haven't identified alternative values in the dye separation literature.